airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -1036,12 +1052,16 @@ protected void assertSameMessages(final List<AirbyteMessage> expected,
         .map(AirbyteMessage::getRecord)
         .peek(recordMessage -> recordMessage.setEmittedAt(null))
         .map(recordMessage -> pruneAirbyteInternalFields ? safePrune(recordMessage) : recordMessage)
-        .map(recordMessage -> recordMessage.getData())
+        .map(AirbyteRecordMessage::getData)
+        .peek(this::sortDataFields)
+        .sorted(Comparator.comparing(JsonNode::toString))
         .collect(Collectors.toList());
 
     final List<JsonNode> actualProcessed = actual.stream()
         .map(recordMessage -> pruneAirbyteInternalFields ? safePrune(recordMessage) : recordMessage)
-        .map(recordMessage -> recordMessage.getData())
+        .map(AirbyteRecordMessage::getData)
+        .peek(this::sortDataFields)
```

Comment:
why is this needed?

Reply:
We sort actual and expected messages for future iterating and assertion of these messages. These lists are sorted by data in string representation. But sometimes data fields for actual and expected message can be sorted in different order, for example:
expected:` "data": { "some_id" : 201, "next_field_name" : "next_field_name_1" }`
actual: `"data": { "next_field_name" : "next_field_name_1", "some_id" : 201 }`
Comparing these strings returns values different from 0 and expected and actual message lists will be sorted in a different order that brings assertion error.` sortDataFields()` fixes this issue by sorting data fields in the same order